manifests/init.pp

```diff
@@ -283,7 +283,7 @@
   String[1] $package_ensure                                      = 'present',
   String[1] $package_name                                        = $redis::params::package_name,
   Stdlib::Absolutepath $pid_file                                 = $redis::params::pid_file,
-  Stdlib::Port::Unprivileged $port                               = 6379,
+  Stdlib::Port $port                                             = 6379,
```

Comment:
Can you update `REFERENCE.md` ?

Reply:
And that's why I hate derived files in git. So easy to forget